app/controllers/customers_controller.rb

```diff
@@ -367,17 +379,19 @@ def send_new_password(email)
       return nil
     end
     begin
-      newpass = String.random_string(6)
-      @customer.password = @customer.password_confirmation = newpass
+      lengthOfToken = 15
```

Comment:
This seems nitpicky, but: please use Ruby style - `camel_case_names` rather than Java-style `snakeCaseNames` - for variable naming.  CodeClimate should have caught this.